Src/VimCore/FSharpUtil.fs

```diff
@@ -559,6 +559,7 @@ module internal CharUtil =
     let IsWhiteSpace x = System.Char.IsWhiteSpace(x)
     let IsNotWhiteSpace x = not (System.Char.IsWhiteSpace(x))
     let IsControl x = System.Char.IsControl x
+    let IsAscii x = x >= 0 && x <= 127

```

Comment:
Believe the type of `x` will be `int` here not `char`.

Reply:
Ha, ha, as much as I love type inference, it does have pitfalls for the unwary!